includes/agentic-commerce/class-wc-stripe-agentic-commerce-order-mapper.php

```
@@ -435,6 +400,101 @@ private function store_stripe_metadata( WC_Order $order, WC_Stripe_Agentic_Check
+				'state'    => $address->get_state() ?? '',
+				'postcode' => $address->get_postal_code() ?? '',
+				'city'     => $address->get_city() ?? '',
+				'address'  => '',
+			],
+			'cart_subtotal'   => 0,
+		];
+
+		$wc_shipping = WC()->shipping();
+
+		if ( ! $wc_shipping instanceof WC_Shipping ) {
+			throw new Exception(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '448', column '14'). (undefined)

(MissingImport)